RichClipboard.package/RCClippingsWindow.class/instance/addClippingsList.st

```diff
@@ -7,5 +7,5 @@ addClippingsList
 		 topFraction: 0;
 		 bottomFraction: 1;
 		 bottomOffset: -1 * self class defaultBottomBarHeight.
-	pluggableListMorph := RCPluggableListMorph newWithClippingsList: self clippingsList.
+	pluggableListMorph := RCPluggableListMorph newWithClippingsList: (self clippingsList) deleteCallback: #removeAt: on: (self clippingsList).
```

Comment:
Würde ich in RCPluggableListMorph einfach hardcoden glaube ich. Oder seht ihr irgendeinen Anwendungsfall wo wir removeAt an ein anderes Objekt senden oder wir sogar eine andere Methode aufrufen?

(Der Konstruktor sollte so sein wie vorher)